### PR TITLE
fix pr-check

### PR DESCRIPTION
**Pull request by BumbleFeng**
Signed-off-by: Feng Huang <fehuang@redhat.com>

### CODE REVIEW

File: dashdotdb/controllers/token.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 }
 
 
-def auth_token(token, required_scopes):
```
Comment (rrati) [inline dashdotdb/controllers/token.py:24]: This is a function signature for oauth and I'm not sure if we can remove the `requires_scopes`.  It's empty unless oauth is used unfortunately.  Can we exclude this from checks?

File: dashdotdb/services/deploymentvalidation.py [hunk 2]
```diff
@@ -34,18 +33,18 @@ def insert(self, token, validation=None):
     def _insert(self, token, item):
         if 'metric' not in item:
             self.log.error('skipping validation: key "metric" not found')
-            return
```
Comment (rrati) [inline dashdotdb/services/deploymentvalidation.py:37]: Make the error message match the log message, so:

return 'key "metric" not found', 400

File: dashdotdb/services/deploymentvalidation.py [hunk 3]
```diff
@@ -34,18 +33,18 @@ def insert(self, token, validation=None):
...
+            return "error", 400
 
         if 'value' not in item:
             self.log.error('skipping validation: key "value" not found')
-            return
+            return 'error', 400
```
Comment (rrati) [inline dashdotdb/services/deploymentvalidation.py:40]: Same here

File: dashdotdb/services/deploymentvalidation.py (hunk 2 above)
Comment (rrati) [reply dashdotdb/services/deploymentvalidation.py:37]: Also, add 400 to the return codes in the swagger.yaml

**Review by jmelis — APPROVED** [review]
LGTM

@rrati we incorporated your comments above, thanks for the review!

**Pull request merged**